client/src/components/global/Footer.vue

```diff
@@ -93,7 +93,7 @@ ccarra1@asu.edu
       &copy; 2020-{{ currentYear }}
       <a href="https://www.azregents.edu/">Arizona Board of Regents</a> |
 
-      <a :href="constants.GITHUB_URL">{{ constants.BUILD_ID }}</a>
+      <a :href="constants.GITHUB_URL">{{ constants.RELEASE_VERSION }}</a>
```

Comment:
for some reason this isn't being rendered properly on staging deployment, shows up as "unknown". Perhaps  the shared/settings environment isn't being set at runtime?

should probably fully merge Constants into `shared/src/settings` as per its FIXME tag

Reply:
oh yeah I think it needs access to these at build time for them to be available. I'll try and do this with vite's support for env vars https://vitejs.dev/guide/env-and-mode